includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1103,18 +1103,39 @@ protected function prepare_payment_information( $order ) {
 	}
 
 	/**
-	 * Update the customer details with the incoming order data.
+	 * Update the customer details with the incoming order data, in a CRON job.
 	 *
 	 * @param int    $order_id       WC order id.
 	 * @param string $customer_id    The customer id to update details for.
-	 */
-	public function update_customer_with_order_data( $order_id, $customer_id ) {
+	 * @param array  $options        Additional options to apply.
+	 */
+	public function update_customer_with_order_data( $order_id, $customer_id, $options = [] ) {
```

Comment:
Nitpick but it might be better to pass the exact params to the async functions instead of an options array here for clear reference in the function signature.

Reply:
Since there are only 2 additional params, this makes sense. Fixed in [9785250](https://github.com/Automattic/woocommerce-payments/pull/4713/commits/9785250eb8f8ebd642102582195d21f78c4619f7).

Reply:
Don't we have to update the accepted params in the `add_action` call for this method for the extra param to be reflected? 🤔  @lovo-h

Reply:
Good catch. Fixed in [f8365a0](https://github.com/Automattic/woocommerce-payments/pull/4713/commits/f8365a0085d6c97ef3f78a1c608605d5a57ceb4c). Also, added a test in [6facffb](https://github.com/Automattic/woocommerce-payments/pull/4713/commits/6facffb68171625ca3aa896d1081ff0eaf69e901), to avoid a similar issue in the future.